pkg/controllers/dlb/controller_test.go

```diff
@@ -97,7 +97,7 @@ func (c *controller) newDaemonSetExpected(rawObj client.Object) *apps.DaemonSet
 							},
 						},
 					},
-					NodeSelector: map[string]string{"kubernetes.io/arch": "amd64"},
```

Comment:
I like this better than using controllers.AddArchConstraintToNodeSelector to be honest.